javase/src/main/java/com/google/zxing/client/j2se/CommandLineEncoder.java

```diff
@@ -19,9 +19,12 @@
 import com.beust.jcommander.JCommander;
 import com.google.zxing.MultiFormatWriter;
 import com.google.zxing.common.BitMatrix;
+import com.google.zxing.EncodeHintType;
 
 import java.nio.file.Paths;
 import java.util.Locale;
+import java.util.Map;
+import java.util.HashMap;
```

Comment:
I'm sorry to nit pick, but while we're here, there are a number of style issues. (Some IDEs can help with this.)

Imports are not ordered alphabetically; still missing a space after arg on line 55; line 58 adds an extra indent that should not be there, and the type of `hints` can really be `Map<EncodeHintType,Object>` as it does not in general only map to strings.